be2-scala/src/main/scala/ch/epfl/pop/pubsub/graph/validators/MeetingValidator.scala

```diff
@@ -38,10 +37,9 @@ sealed class MeetingValidator(dbActorRef: => AskableActorRef) extends MessageDat
 
         val sender: PublicKey = message.sender
         val channel: Channel = rpcMessage.getParamsChannel
-        if (!data.end.isDefined)
-          Right(validationError("data.end is not defined"))
 
         runChecks(
+          checkDataEndIsDefined(rpcMessage, data.end, validationError("data.end is not defined")),
```

Comment:
I don't think this behavior is correct, from the protocol spec for [Meeting#Create](https://github.com/dedis/popstellar/blob/master/docs/messageData.md#creating-a-meeting-meetingcreate)
the end of the meeting is optional so an error should only be thrown if an end is defined but invalid (e.g the end is before the beginning)

Reply:
Nice comment. Thank you for reminding us the protocol (forgot it as well oops).

Reply:
Okey, I get it, so the idea would be to run the checks concerning data.end only in the case where data.end is defined, if it is not, then no need to run these checks since they will throw an exception ?

Reply:
Exactly :)